apis/r/inst/AUTHORS

```diff
@@ -0,0 +1,77 @@
+
+The 'tiledbsoma' R package has been written by a team comprised of
+members from Chan Zuckerberg Initiative (CZI) and TileDB, Inc., as
+part of the 'SOMA' ("Stack of Matrices, Annotated") initative.
+
+The repository provides detailed commit statistics across the C++,
+Python, and R components of the implementation:
+  https://github.com/single-cell-data/TileDB-SOMA/graphs/contributors
+
+The package also includes code written by other contributors as
+detailed below:
+
+
+-- libtiledbsoma/src/external/include/span/span.hpp ---------------------------------------------
+The span.hpp implements std::span, part of C++20, for use by C++11 or later. It
+was written by Tristan Brindle and is released at https://github.com/tcbrindle/span
+
+//          Copyright Tristan Brindle 2018.
+// Distributed under the Boost Software License, Version 1.0.
+//    (See accompanying file ../../LICENSE_1_0.txt or copy at
+//          https://www.boost.org/LICENSE_1_0.txt)
+
+
+-- libtiledbsoma/src/external/{src,include}/thread_pool/ ----------------------------------------
+The thread_pool implementation is from TileDB, Inc., and part of TileDB Embedded released at
+https://github.com/tiledb-inc/tiledb
+
+ * @copyright Copyright (c) 2017-2022 TileDB, Inc.
+ *            Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+
+ * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * Neither the name of Google Inc. nor the names of its contributors may be used
+ * to endorse or promote products derived from this software without specific
+ * prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+-- apis/r/src/nanaarrow.{h,cc}/ -------------------------------------------------------------
```

Comment:
Typo: nanaarrow -> nanoarrow

Reply:
Thanks for spotting this, fixed.  But ... (read on)